.github/workflows/release-embedding-sdk.yml

```diff
@@ -86,9 +84,23 @@ jobs:
         with:
           m2-cache-key: "release-sdk"
 
+      - name: Update readme
```

Comment:
This extra step is crucial because we're now not putting the most updated changelog in the SDK. You can see our [NPM package page](https://www.npmjs.com/package/@metabase/embedding-sdk-react?activeTab=readme) that the version 0.1.5 still refers to 0.1.4 in the readme

You can verify that this is fixed by downloading `metabase-sdk` artifact [from this build](https://github.com/metabase/metabase/actions/runs/9317795703). And check the readme file that it should refer to the correct version. Although it's not that obvious at this moment because we just change all the version reference in SDK readme to 50 RC. So now the SDK readme won't be updated after every SDK release anymore.